test/README.md

```diff
@@ -0,0 +1,6 @@
+# Adding new test
+
+Add a test file in the [Test folder](emlib/test/tests).
```

Comment:
This looks like a clickable url-link. Change to "test folder `emlib/test/tests`? :)